manifests/init.pp

```diff
@@ -99,10 +108,12 @@
   Boolean $fwd_conntrack = false,
   Boolean $inet_filter = true,
   Boolean $nat = true,
+  Boolean $allow_unmanaged_rules = true,
   Hash $rules = {},
   Hash $sets = {},
   String $log_prefix = '[nftables] %<chain>s %<comment>s',
   String[1] $nat_table_name = 'nat',
+  Stdlib::Unixpath $inmem_rules_hash_file = '/var/cache/nft-memhash',
```

Comment:
Wouldn't something in `/run` or `/var/run` make more sense?

Reply:
Maybe, I was not sure about this :)

Reply:
It defaults now to `/run/puppet-nft-memhash`.